.github/CODEOWNERS

```diff
@@ -112,4 +112,8 @@ ISSUE_TRIAGE_PROCESS.rst @eladkal
 /providers/fab/src/airflow-core/src/airflow/providers/fab/migrations/ @ephraimbuddy
 
 # AIP-72 - Task SDK
+# Python SDK
 /task-sdk/ @ashb @kaxil @amoghrajesh
+
+# Golang SDK
+/go-sdk/ @ashb @kaxil @amoghrajesh
```

Comment:
The folder structure will be `{language}-sdk/` ?
I assume it won't be long before we have several. Do we want to consider `sdk/{language}/` structure?

Reply:
There were comments about this on ash's initial PR. Its a parallel discussion which we can have